lang/de.po

```diff
@@ -0,0 +1,40 @@
+msgid ""
+msgstr ""
+"Language: de\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+
+msgid "msg_help_usage"
+msgstr "Verwendung: vrms-rpm [Optionen]"
+msgid "msg_help_option_ascii"    
+msgstr "    Display rms ASCII-art wenn keine kommerziellen Pakete gefunden wurden\n"
+       "    oder wenn kommerzielle Pakete 10%% oder mehr der gesamten Pakete ausmachen."
+msgid "msg_help_option_explain"
+msgstr "    Bei den gelisteten Paketen sind die gezeigten Lizenzen da, um kommerzielle\n"
+       "    und freie Pakete zu klasifizieren."
+msgid "msg_help_option_help"
+msgstr "    Zeigt diese Hilfe an und beendet das Programm."
+msgid "msg_help_option_list"
+msgstr "    Abgesehen von der Anzeige einer zusammenfassenden Anzahl von freien & kommerziellen\"
```

Comment:
The lines in this message must not exceed 80 characters. Break the message into multiple lines using `\n`.